draft-ietf-taps-interface.md

```diff
@@ -644,10 +644,8 @@ indicates a preference for a specific path by specifying an interface, but also
 preference for a protocol not available on this path, the transport system will
 try the path first, ignoring the preference.
 
-Both Selection and Connection Properties can be added to a Preconnection to
-configure the selection process, and to further configure the eventually
-selected protocol stack(s). They are collected into a TransportProperties object
-to be passed into a Preconnection object:
+Selection, and Connection Properties, as well as defaults for Message Properties, can be added to a Preconnection to configure the selection process, and to further configure the eventually selected protocol stack(s).
+They are collected into a TransportProperties object to be passed into a Preconnection object:
 
 ~~~
 TransportProperties := NewTransportProperties()
```

Comment:
I would add an extra sentence here to say, e.g.: "For protocols that do not protect against duplicated messages, e.g., UDP, a default Message Property of Idempotent=enabled must be assigned."

Reply:
I thought about it, but decided against putting it there. This is a general section, and does not describe special cases. We may add it as an example here, but still looks like an odd place for this.

Reply:
That works for me

Reply:
I agree with @philsbln that it does not fit to add such details here.

Reply:
Should you be able to change the default message properties after connection establishment? Right now you would not be able to as the preconnection gets consumed.

Reply:
I'm not sure this is necessary to specify. Clearly a connection can change the per-message properties at any time - but I suggest that changing the default established in the preconnection adds more  text which isn't strictly necessary. Omitting this does not prevent an API providing it as an additional feature, but then it would then need to work out what to do when the connection is cloned and preconnection properties are replicated. I'd rather not delve there.

Reply:
@MaxF12 This is possible by changing the properties on the Connection object. Doing it on the Preconnection would be harmful, as we want to make the preconnection reusable. If you think this needs discussion, please open a new Issue for that.